Jellyfin.Plugins.PushOverNotifications/Api/ServerApiEntryPoints.cs

```diff
@@ -23,9 +22,9 @@ class ServerApiEndpoints : IService
         private readonly IHttpClient _httpClient;
         private readonly ILogger _logger;
 
-        public ServerApiEndpoints(ILogManager logManager, IHttpClient httpClient)
+        public ServerApiEndpoints(ILoggerFactory logManager, IHttpClient httpClient)
```

Comment:
In this case I would probably use the logger that the plugin creates instead

Reply:
Sounds good to me :P

Reply:
I tried to do what you suggested for the this and the one below, but just ended up breaking the plugin due to my limited knowledge of C#

Reply:
When you say "logger that the plugin creates" what are we talking about? I'm not sure if I've done what you wanted or now XD.